src/FixedDataTableContainer.js

```diff
@@ -150,8 +150,8 @@ class FixedDataTableContainer extends React.Component {
   static getBoundState(reduxStore) {
     const state = reduxStore.getState();
     const boundState = pick(state, [
-      'columnGroupProps',
-      'columnProps',
+      'columnGroupElements',
+      'columnElements',
```

Comment:
`state.columnProps` has been replaced with `state.columnElements`.

One reason being that there's a modified version of `columnProps` returned by selectors, leading to confusion between `state.columnProps` and `someSelector().columnProps`.